frontend/src/lib/components/AddInsightsToDashboard/AddInsightsToDashboard.tsx

```diff
@@ -0,0 +1,28 @@
+import { LemonButton } from '@posthog/lemon-ui'
+
+import { DashboardType } from '~/types'
+
+interface AddInsightsToDashboardProps {
+    setAddInsightsToDashboardModalOpen: (open: boolean) => void
+    dashboard: DashboardType
+    disabledReason: string | null
+}
+
+export function AddInsightsToDashboard({
+    setAddInsightsToDashboardModalOpen,
+    dashboard,
+    disabledReason,
+}: AddInsightsToDashboardProps): JSX.Element | null {
+    return (
+        <LemonButton
```

Comment:
This button previously had a dropdown menu that allows adding a text card to a dashboard.

<img width="189" alt="Screenshot 2024-01-19 at 13 12 58" src="https://github.com/PostHog/posthog/assets/1851359/a46363ac-0c56-478e-b7d6-dfc709a881af">